clean_project/clean_file.py

```diff
@@ -16,6 +16,7 @@ def clean_up(src_path, dest_path):
         with codecs.open(dest_path, 'w', 'utf-8') as dest_file:
             for line in source_file:
                 dest_file.write(sanitize(line))
+    return dest_file.name
```

Comment:
why do you need to return this path here? and anyway, you should return dest_path